Mathlib/Algebra/Order/Hom/Monoid.lean

```diff
@@ -54,7 +52,7 @@ making some definitions and lemmas irrelevant.
 
 ## Tags
 
-ordered monoid, ordered group, monoid with zero
+ordered monoid, ordered group
 -/
 
 
```

Comment:
This LGTM; not sure about what assert would be best, maybe this?
```suggestion
assert_not_imported Mathlib.Algebra.Order.GroupWithZero.Canonical
```

bors d=@YaelDillies